src/objects/Line.js

```diff
@@ -74,6 +76,38 @@ class Line extends Object3D {
 
 	}
 
+	_raycastSegment( iStart, iEnd, raycaster, ray, localThresholdSq ) {
```

Comment:
Instead of making a private method, can you please turn this into a module scope function and rename it to `checkIntersection()` like in `Mesh`?

It's best if you add the `object` to the signature. In this way, you have access to the geometry and world matrix.

Reply:
Thanks for the reply and for making the demonstration.

After comparing with the raycast computation methods in Mesh.js, I think maybe change the signature of this function into the following for better consistency:

```js
function checkIntersection( object, raycaster, ray, thresholdSq, a, b )
// turn into a module scope function and rename it to `checkIntersection()`
// add `object` as a parameter
// changed order of parameters
// renamed `iStart, iEnd` to `a, b`
// rename `localThresholdSq` to `thresholdSq`
```